Gopkg.toml

```diff
@@ -44,17 +44,18 @@ required = [
   name = "k8s.io/client-go"
   version = "kubernetes-1.11.3"
 
+[[constraint]]
+  name = "sigs.k8s.io/controller-tools"
+  source = "https://github.com/n3wscott/controller-tools.git"
```

Comment:
is there a reason why this is forked? If so, is there a path forward where we can use the non-forked version? Perhaps a comment / issue to track this would be in order. Just to make sure that we're not taking on something that will be forked forevs.

Reply:
added a comment.

Reply:
Is there an issue tracking this in the kubebuilder repo?